app/views/merchant_details/merchant_details.njk

```diff
@@ -0,0 +1,80 @@
+{% extends "../layout.njk" %}
+
+{% block page_title %}
+  Organisation details - {{currentService.name}} {{currentGatewayAccount.full_type}} - GOV.UK Pay
+{% endblock %}
+
+{% block mainContent %}
+<div class="column-full">
+  <a href="/my-services" class="link-back">Back to services</a>
+</div>
+<div class="column-two-thirds">
+  <h1 class="page-title">Organisation details</h1>
+  {% if has_direct_debit_gateway_account %}
```

Comment:
This logic will create problems. Let's say that a service as both a card and dd gateway account, they will see all of three descriptions (the dd, card, and also the "has both accounts" text), which is not great. 
A solution would be to move the `{% if has_card_and_dd_gateway_account %}` first, and use else if for the subsequent two conditions